src/domain/entities/TranslatableText.ts

```diff
@@ -16,3 +17,24 @@ export const buildTranslate = (locale: string): TranslateMethod => {
 };
 
 export type TranslateMethod = (string: TranslatableText) => string;
+
+export const importTranslate = <T extends TranslatableText>(item: T, language: string, term: string | undefined): T => {
+    if (term === undefined) {
+        return item;
+    } else if (language === "en") {
+        return { ...item, referenceValue: term };
+    } else {
+        return { ...item, translations: { ...item.translations, [language]: term } };
```

Comment:
Personal opinion: I often prefer to not use `else` when we are returning on the `if` statement but @tokland tells me to use `if/else` branches so I would say it is good like this